app/models/test_track/remote/split_registry.rb

```diff
@@ -15,7 +15,7 @@ def instance
       if faked?
         new(fake_instance_attributes(nil))
       else
-        get('/api/v2/split_registry')
+        get("/api/v3/builds/#{TestTrack::BuildTimestamp}/split_registry")
```

Comment:
We may be able to pass in the timestamp as an argument to `get`. Though we don’t need url encoding since the timestamp shouldn’t have any funky characters, I think it’s a good practice.

Reply:
Seems like we're doing string interpolation in the other instances of get with args. Should we stick with that pattern or forge our own path?